app/views/triggeredMigration/CheckHmrcRecordsView.scala.html

```diff
@@ -192,7 +222,9 @@ <h2 class="govuk-summary-card__title" id="foreign-property-heading">
     @soleTraderSection
     @h3(messages("triggered-migration.check-hmrc-records.property.heading"), classes = "govuk-heading-m govuk-!-margin-top-5", optId = Some("property-heading"))
     @propertyBusinessSection
-
+    @if(ceaseBannerParams._2 != 0) {
+        @ceasedBusinessSection
+    }
```

Comment:
There's a missing seperator between the ceased business heading and the property businesses section